code/02_create-figures.R

```diff
@@ -0,0 +1,280 @@
+#########################################################################
+# Name of file - 02_create-figures.R
+# Data release - End of Life Publication
+# Original Authors - Alice Byers
+# Orginal Date - September 2019
+#
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+#
+# Description - Create and save figures for summary and report.
+#
+# Approximate run time - xx minutes
+#########################################################################
+
+
+### 1 - Setup environment and load functions ----
+
+source(here::here("code", "00_setup-environment.R"))
+source(here::here("functions", "summarise_data.R"))
+source(here::here("functions", "calculate_qom.R"))
+
+
+### 2 - Read in basefile ----
+
+basefile <- read_rds(here("data", "basefiles", 
+                          glue("{pub_date}_base-file.rds")))
+
+
+### 3 - Figure 1 - Trend Bar Chart ----
+
+fig1 <- 
+  
+  basefile %>%
+  group_by(fy) %>%
+  summarise(qom_hosp = calculate_qom(sum(los), sum(deaths), "hosp"),
+            qom_comm = calculate_qom(sum(los), sum(deaths), "comm")) %>%
```

Comment:
Shall we use excel_data you created in "03_create-excel-tables.R" and filter out what we need for the plots? For example, fig1 will filter out category == "Scotland". Then you can continue with pivot_longer etc to plot the chart. 

So as for the other plots? If we do that, we may need to adjust the order of the scripts. e.g. 02_create-aggregated-file; 03_create-figures.

Reply:
Hi Tina, I've tried creating the figures using `excel_data` this morning, but have decided that it will be easier to just use the `summarise_data()` function. Using `excel data` involved a wee bit extra code and also would've required a bit of work for the age/gender figure as for excel these are combined and include totals for each. Hope that makes sense but happy to discuss further. 

I think `excel_data` will be useful for creating the open data file as I'm thinking these files will follow a similar format.